packages/loader/container-loader/src/connectionManager.ts

```diff
@@ -613,6 +614,14 @@ export class ConnectionManager implements IConnectionManager {
 				// NOTE: This isn't strictly true for drivers that don't require network (e.g. local driver).  Really this logic
 				// should probably live in the driver.
 				await waitForOnline();
+				this.logger.sendTelemetryEvent({
```

Comment:
I'd think this should be performance event, and I'd also add retryDelayFromError  as a property on it.

Reply:
Changed it to perf event. Also, I logged these props in details props alongside delayMS as such properties are now logged in details.